docs/_docs/storage/persistent.adoc

```diff
@@ -0,0 +1,90 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+= Persistent Storage
+
+//:javaFile: {javaCodeDir}/IgnitePersistence.java
+
+== Overview
+
+Ignite Persistence is designed to provide a quick and responsive persistent storage.
+When it is enabled for the link:config/data-region[data region], Ignite stores all the data on disk, and loads as much data as it can into RAM for processing.
+
+The Native Persistence functionality is based on the following features:
+
+* Storing data partitions on disk
+* Checkpointing
+* Usage of OS swap
+
+When persistence is enabled, Apache Ignite stores each partition in a separate file on disk. In addition to data partitions, Apache Ignite stores indexes and metadata.
+
+//image::images/persistent_store_structure.png[]
+
+== Enabling Persistent Storage
+
+Native Persistence is configured per link:config/data-region[data region].
+To enable native persistent storage, set the data region to `aipersist`.
```

Comment:
This is not true again. Each engine has independent collection of data regions, thus you can't say that we assign an engine to the region, it's the opposite